sessions/js-functions-2/greeting-page/js/index.js

```diff
@@ -0,0 +1,33 @@
+console.clear();
+
+/*
+Change the contents of this page depending on the current day and time.
+
+- Write a function getGreeting that returns a different greeting depending on the current time:
+  - 6-12am: returns "Good Morning"
+  - 12am - 6pm: returns "Good Afternoon"
+  - 6pm - 10pm: returns "Good Evening"
+  - 10pm - 6am: returns "Good Night"
```

Comment:
12am is midnight and 12pm is noon - which is confusing and used the other way around here. It might be better to use a 24 hrs system to describe the exercise.